internal/pkg/cli/svc_init.go

```diff
@@ -376,11 +377,11 @@ func (o *initSvcOpts) askImage() error {
 		return nil
 	}
 
-	var validator prompt.ValidatorFunc
+	validator := validateNotDockerfile
 	promptHelp := wkldInitImagePromptHelp
 	if o.wkldType == manifest.RequestDrivenWebServiceType {
 		promptHelp = wkldInitAppRunnerImagePromptHelp
-		validator = validateAppRunnerImage
+		validator = chainValidators(validator, validateAppRunnerImage)
```

Comment:
Why not
```suggestion
		validator = chainValidators(validateNotDockerfile, validateAppRunnerImage)
```
🤔

Reply:
It doesn't matter anymore, but I think the change above is cleaner because it's like Go's `append` function where you can keep adding more validators on top of each other